python/whylogs/migration/converters.py

```diff
@@ -106,51 +107,80 @@ def _generate_segment_tags_metadata(
     return segment_message_tags, segment_tags, segment_metadata
 
 
-def read_v0_to_view(path: str) -> DatasetProfileView:
+def read_v0_to_view(path: str, allow_partial: bool = True) -> DatasetProfileView:
```

Comment:
allow partial defaults to true here, but in `v0_to_v1_view` it defaults to false.
Should we set both to the same value. I'd vote for the default to be False, and in the example we could set it to True and explain.

Reply:
I was on the fence there, but sure the read_v0_to_view can make callers pass in allow_partial = True and raise an exception by default on errors, thanks for the comment, updated.